src/url.ts

```diff
@@ -19,6 +19,28 @@ export function parseChannelFromURL() {
   return [channelID, channelLogin] as const;
 }
 
+
+export const fadeout= resolveFadeout();
+function resolveFadeout() {
+  const raw = params.get("fadeout");
+  if (raw === "off" || raw === "none") {
+    return null;
+  }
+
+  if (!raw) {
+    return "15s";
+  }
```

Comment:
do you need this? wouldn't `isNaN` return true for `""`

Reply:
Yeah technically it would work without it. But I have it because parseInt only takes string as param and raw it type string/null